app/views/reports/_form.html.slim

```diff
@@ -0,0 +1,10 @@
+h1 Student Assessment Form
+= simple_form_for @report, html: { class: 'form' } do |f|
+  = f.input :student_id, collection: @students.map { |x| [x.name, x.id] }
+  - %i(participation effort skill overall).each do |metric|
+    = f.input metric, collection: 1..3, as: :radio_buttons, checked: 3, wrapper_html: { class: 'col-sm-7' }, label_html: { class: 'radio-inline col-sm-3' }, item_wrapper_class: 'radio-inline'
+    = f.input "#{metric}_comments".to_sym, as: :text , wrapper_html: { class: 'col-md-5' }
+  = f.input :status, selected: 0
```

Comment:
It looks like we lost the ability to generate a report for a student who isn't in the system. That seems kind of weird, I know, but one of the main drives behind building it that way (and why the controller action was grabbing everything from the params hash) was so other instructors who are not using the app can still generate reports for their students. Aaron was having his grading app use the report generation as an api to get reports and Sam can input her students manually, which is still much faster than the way we've been doing it with pdf editors.